components/toggle_modal_button_redux/toggle_modal_button_redux.jsx

```diff
@@ -9,7 +9,7 @@ import {intlShape} from 'utils/react_intl';
 
 class ModalToggleButtonRedux extends React.PureComponent {
     static propTypes = {
-        accessibilityLabel: PropTypes.string,
+        accessibilityLabel: PropTypes.node,
```

Comment:
Is this okay to change? We pass this into an aria-label, so it might need to be a string

Reply:
No it's not, and thanks for catching. I believe it was a "correction" to a console error. Should've dug deeper... I think there's place higher up that accepts a React.ReactNode and eventually passes to it accessibilityLabel. I'll try to find it, or revert this if I can't.

Reply:
reverted